bears/natural_language/LanguageToolBear.py

```diff
@@ -25,12 +25,7 @@ def check_prerequisites(cls):
         if shutil.which('java') is None:
             return 'java is not installed.'
         else:
-            try:
-                from language_check import LanguageTool, correct
```

Comment:
Mixih is correct. This cant be deleted. It needs to be modified so that pyflakes believes it is "used".